oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/SearchAppFacets.jsx

```diff
@@ -15,7 +16,12 @@ export const SearchAppFacets = ({ aggs, appName, allVersionsToggle }) => {
           />
         )}
         {aggs.map((agg) => (
-          <BucketAggregation key={agg.aggName} title={agg.title} agg={agg} />
+          <ErrorBoundary
+            fallback={<div>{i18next.t("Error loading facet")}</div>}
```

Comment:
Inlcude facet title in error message

Reply:
![Screenshot 2025-05-09 at 18 41 16](https://github.com/user-attachments/assets/c971c1ef-adf5-44f1-a81d-ee34e60c3da9)

show better fallback error block (like for list item)